src/encoded/static/scss/encoded/modules/_panels.scss

```diff
@@ -496,6 +496,31 @@ $file-gallery-controls-height: 34px; // Heit of controls inside file-gallery-con
             height: $file-gallery-controls-height;
             vertical-align: top;
         }
+
+        @media screen and (max-width: $screen-sm) {
```

Comment:
Need to reverse the logic and use `min-width` to align with all the other uses of mobile-first that we inherited from Bootstrap. Could use the more common `$screen-sm-min` as a reminder that these should always be `min-width`. Mixing `min-width` and `max-width` will make maintenance a nightmare.

Reply:
Thanks for this feedback. It was pretty helpful.